### PR TITLE
test pr

### PR DESCRIPTION
**Pull request by isqua-test**

### CODE REVIEW

File: src/form.js [hunk 1]
```diff
@@ -2,171 +2,169 @@
...
+var rating = reviewForm.elements['review-mark'];
+
+var nameField = reviewForm.elements['review-name'];
+var feedbackField = reviewForm.elements['review-text'];
+
+var hintsContainer = reviewForm.querySelector('.review-fields');
+var hintName = hintsContainer.querySelector('.review-fields-name');
+var hintFeedback = hintsContainer.querySelector('.review-fields-text');
+
+var submitBtn = reviewForm.querySelector('.review-submit');
```
Comment (isqua-test) [inline src/form.js]: test

File: src/load.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+// представляющим собой реестр функций
+window.CallbackRegistry = {};
+
+  /**
+   * Получает данные с сервера по JSONP
+   * по указанному http-запросу
+   * @param {String} url: адрес, по которому надо получить данные
+   * @param {Function} cb: callback-функция, в которая получит данные после загрузки
+   */
+module.exports = function(url, cb) {
```
Comment (isqua-test) [inline src/load.js]: test

Comment (isqua) [inline src/load.js]: test

Comment (isqua) [inline src/load.js]: test comment on line 12 in file load.js

File: src/form.js [hunk 3]
```diff
@@ -2,171 +2,169 @@
...
 
-  var nameField = reviewForm.elements['review-name'];
-  var feedbackField = reviewForm.elements['review-text'];
+  var isFeedbackValid = !feedbackField.required || feedbackField.value;
 
-  var hintsContainer = reviewForm.querySelector('.review-fields');
-  var hintName = hintsContainer.querySelector('.review-fields-name');
-  var hintFeedback = hintsContainer.querySelector('.review-fields-text');
+  /**
+   * убираем подсказку в блоке hintsContainer (или блок целиком) в зависимости от
```
Comment (isqua-test) [inline src/form.js]: test

File: src/load.js (hunk 2 above)
Comment (isqua-test) [inline src/load.js]: test comment on line 12 in src/load.js

Comment (isqua-test) [inline src/load.js]: why it is on line 10 now?

Comment (staceb) [inline src/load.js]: Test comment on line 12 - staceb

File: src/load.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  // число приводится к строке, последние 6 символов которой записываются в переменную в качестве имени функции
+  var callbackName = 'cb' + String(Math.random()).slice(-6);
+
+  /**
+   * Результат вызова JSONP-скрипта, с данными в качестве параметра
+   * @param {Object[]} data: {*}
+   */
+  window.CallbackRegistry[callbackName] = function(data) {
+    // cb — функция, в которую надо передать данные
+    // data – данные, которые нужно передать в функцию
```
Comment (staceb) [inline src/load.js]: Test to see if its only specific to line 12

File: src/load.js (hunk 2 above)
Comment (isqua) [inline src/load.js]: @staceb remove your comment and try again comment this string from Files tab.

File: src/form.js [hunk 5]
```diff
@@ -2,171 +2,169 @@
...
+ * Cookies
+ */
+
+/**
+ * функция, возвращающая кол-во дней, прошедших с указанной даты
+ * @param {Number} month месяц, от 0 до 11
+ * @param  {Number} date дата, от 1 до 31
+ * @returns {Number}
+ */
+function getExpiresDate(month, date) {
```
Comment (isqua) [inline src/form.js]: comment line number 95

File: src/game.js [hunk 6]
```diff
@@ -1,826 +1,824 @@
...
+  }
+};
+
+/**
+ * Обновление движения файрбола. Файрбол выпускается в определенном направлении
+ * и после этого неуправляемо движется по прямой в заданном направлении. Если
+ * он пролетает весь экран насквозь, он исчезает.
+ * @param {Object} object
+ * @param {Object} state
+ * @param {number} timeframe
```
Comment (isqua) [inline src/game.js]: comment 154

Comment (isqua) [inline src/game.js]: It’s on line 151 now :(

File: src/game.js [hunk 7]
```diff
@@ -1,826 +1,824 @@
...
+ */
+var LevelsRules = {};
+
+/**
+ * Уровень считается пройденным, если был выпущен файлболл и он улетел
+ * за экран.
+ * @param {Object} state
+ * @return {Verdict}
+ */
+LevelsRules[Level.INTRO] = function(state) {
```
Comment (isqua) [inline src/game.js]: Comment line 199

File: src/game.js [hunk 8]
```diff
@@ -1,826 +1,824 @@
...
+      'intro': 'Я умею перемещаться и летать по нажатию на стрелки. А если нажать Shift, я выстрелю файерволом! И помни: предмет деятельности амбивалентно творит сенсибельный гравитационный парадокс'
+    };
+
+    var message;
+    switch (this.state.currentStatus) {
+      case Verdict.WIN:
+        message = messagesList.win;
+        break;
+
+      case Verdict.FAIL:
```
Comment (isqua) [inline src/game.js]: Comment 4141

File: src/load.js (hunk 2 above)
Comment (francisfuzz) [inline src/load.js]: Test Comment on line 12 - francisfuzz

**Comment by isqua-test** [comment]
isqua isqua test test

**Comment by isqua-test** [comment]
isqua isqua test test

**Pull request merged**